components/infobox/wikis/rainbowsix/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,248 @@
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Team = require('Module:Team')
+local Variables = require('Module:Variables')
+local Template = require('Module:Template')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Builder = require('Module:Infobox/Widget/Builder')
+
+local _BANNED = mw.loadData('Module:Banned')
+local _ROLES = {
+	-- Players
+	['entry'] = {category = 'Entry fraggers', variable = 'Entry fragger', isplayer = true},
+	['support'] = {category = 'Support players', variable = 'Support', isplayer = true},
+	['flex'] = {category = 'Flex players', variable = 'Flex', isplayer = true},
+	['igl'] = {category = 'In-game leaders', variable = 'In-game leader', isplayer = true},
+
+	-- Staff and Talents
+	['analyst'] = {category = 'Analysts', variable = 'Analyst', isplayer = false},
+	['observer'] = {category = 'Observers', variable = 'Observer', isplayer = false},
+	['host'] = {category = 'Hosts', variable = 'Host', isplayer = false},
+	['journalist'] = {category = 'Journalists', variable = 'Journalist', isplayer = false},
+	['expert'] = {category = 'Experts', variable = 'Expert', isplayer = false},
+	['coach'] = {category = 'Coaches', variable = 'Coach', isplayer = false},
+	['caster'] = {category = 'Casters', variable = 'Caster', isplayer = false},
+	['talent'] = {category = 'Talents', variable = 'Talent', isplayer = false},
+	['manager'] = {category = 'Managers', variable = 'Manager', isplayer = false},
+	['producer'] = {category = 'Producers', variable = 'Producer', isplayer = false},
+	['admin'] = {category = 'Admins', variable = 'Admin', isplayer = false},
+}
+_ROLES.entryfragger = _ROLES.entry
+_ROLES['assistant coach'] = _ROLES.coach
+
+local _GAMES = {
+	r6s = '[[Rainbow Six Siege|Siege]]',
+	vegas2 = '[[Rainbow Six Vegas 2|Vegas 2]]',
+}
+_GAMES.siege = _GAMES.r6s
+
+local _SIZE_OPERATOR = '25x25px'
+
+local CustomPlayer = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+function CustomPlayer.run(frame)
+	local player = Player(frame)
+
+	if String.isEmpty(player.args.team) then
+		player.args.team = PlayerTeamAuto._main{}
+	end
+
+	if String.isEmpty(player.args.team2) then
+		player.args.team2 = PlayerTeamAuto._main{team = 'team2'}
+	end
+
+	player.args.history = Template.safeExpand(frame, 'TeamHistoryAuto', {player = player.args.id})
+
+	player.adjustLPDB = CustomPlayer.adjustLPDB
+	player.createBottomContent = CustomPlayer.createBottomContent
+	player.createWidgetInjector = CustomPlayer.createWidgetInjector
+	player.defineCustomPageVariables = CustomPlayer.defineCustomPageVariables
+
+	_args = player.args
+
+	return player:createInfobox(frame)
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'status' then
+		return {
+			Cell{name = 'Status', content = CustomPlayer._getStatusContents()},
+			Cell{name = 'Years Active (Player)', content = {_args.years_active}},
```

Comment:
fyi there exists a module to automate that

Reply:
Leaving it out for now